models/lido/accounting/ethereum/lido_accounting_ethereum_buffer_inflow.sql

```diff
@@ -0,0 +1,27 @@
+{{ config(
+        alias ='buffer_inflow',
+        partition_by = ['period'],
```

Comment:
if i'm reading the spell correctly, this is partitioning by `evt_block_time`? that is much too granular for partitions. i'd suggest either removing this config option or introducing a new column in the select below that is at minimum day (or week, month, etc) to partition by

Reply:
thank you for review, fixed config with new "day" column

Reply:
great! we'll let the test run, then hopefully all set

Reply:
It seems that test execution takes too long and fails after 90 minutes. @jeff-dude, maybe removing partition will help here or it is something in test flow?

Reply:
i don't think it's due to your spells in the PR. for some reason, our gh action ci test is picking up more models than it should and causing longer runtimes, then timeouts. i'm trying to look into that now. i would assume no changes here, just waiting for resolution there and a successful run after

Reply:
all good, unrelated issue resolved